Taggers/python/flashggTags_cff.py

```diff
@@ -435,87 +435,76 @@
-                                   deltaRMuonPhoThreshold = cms.double(0.5),
-                                   jetsNumberThreshold = cms.double(3.),
-                                   jetPtThreshold = cms.double(20.),
-                                   jetEtaThreshold= cms.double(2.4),
-                                   deltaRPhoLeadJet = cms.double(0.4),
-                                   deltaRPhoSubLeadJet = cms.double(0.4),
-                                   muPFIsoSumRelThreshold = cms.double(0.25),
-                                   deltaRJetMuonThreshold = cms.double(0.4),
-                                   PuIDCutoffThreshold = cms.double(0.8),
-                                   PhoMVAThreshold = cms.double(-0.9),
-                                   METThreshold = cms.double(45.),
-                                   DeltaRTrkElec = cms.double(.4),
-                                   TransverseImpactParam = cms.double(0.02),
-                                   LongitudinalImpactParam = cms.double(0.2),
-                                   deltaRPhoElectronThreshold = cms.double(1.),
-                                   deltaMassElectronZThreshold = cms.double(10.),
-                                   electronEtaThresholds=cms.vdouble(1.4442,1.566,2.5),
-                                   nonTrigMVAThresholds = cms.vdouble(0.913286,0.805013,0.358969),
-                                   nonTrigMVAEtaCuts = cms.vdouble(0.8,1.479,2.5),
-                                   electronIsoThreshold = cms.double(0.15),
-                                   electronNumOfHitsThreshold = cms.double(1),
-                                   useElectronMVARecipe = cms.bool(False),
-                                   useElectronLooseID = cms.bool(True)
+                                    DiPhotonTag                 = cms.InputTag('flashggPreselectedDiPhotons'),
+                                    SystLabel                   = cms.string(""),
+                                    ElectronTag                 = cms.InputTag('flashggSelectedElectrons'),
+                                    MuonTag                     = cms.InputTag('flashggSelectedMuons'),
+                                    inputTagJets                = UnpackedJetCollectionVInputTag,
+                                    METTag                      = cms.InputTag('flashggMets'),
+                                    VertexTag                   = cms.InputTag('offlineSlimmedPrimaryVertices'),
+                                    rhoTag                      = cms.InputTag('fixedGridRhoFastjetAll'),
+                                    MVAResultTag                = cms.InputTag('flashggDiPhotonMVA'),
+                                    GenParticleTag              = cms.InputTag( "flashggPrunedGenParticles" ),
+                                    WHMVAweightfile             = cms.FileInPath("flashgg/Taggers/data/TMVA_WHLeptonicTag_BDT_16Dec2019.weights.xml"),
```

Comment:
Is this file common to all years? Is it already on EOS?

Reply:
Yes, it's for full runII and is put in eos.